examples/hello.py

```diff
@@ -1,19 +1,20 @@
 # hello
-# Built with Seahorse v0.2.3
+# Built with Seahorse v0.2.0
 #
 # Greets users to Solana by printing a message and minting them a token!
 
 from seahorse.prelude import *
 
-declare_id('Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS')
+declare_id('')
```

Comment:
I'd rather leave a non-empty ID in the examples, because you need to put one when you build locally. The empty one will only work in Playground, which will also replace a non-empty one.